packages/attribute-registry-consumer/Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY pnpm-workspace.yaml /app/
 COPY ./packages/attribute-registry-consumer/package.json /app/packages/attribute-registry-consumer/package.json
 COPY ./packages/commons/package.json /app/packages/commons/package.json
 COPY ./packages/models/package.json /app/packages/models/package.json
+COPY ./packages/kafka-iam-auth/package.json /app/packages/kafka-iam-auth/package.json
```

Comment:
not a blocker for this PR, but I wander if there is a better way to include dependencies in the image, since they are already defined in the TS project